src/SDKs/CognitiveServices/dataPlane/Search/BingVisualSearch/BingVisualSearch/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\..\..\tools\generate.cmd cognitiveservices/data-plane/VisualSearch %*
```

Comment:
Please create a generate.ps1 file similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/generate.ps1) and regenerate the code